_search-plugins/hybrid-search.md

```diff
@@ -569,4 +569,454 @@ The response contains the matching documents and the aggregation results:
     }
   }
 }
+```
+
+## Hybrid search with sorting
+**Introduced 2.16**
+{: .label .label-purple }
+
+By default, the standard hybrid search without sorting returns search results that are ordered by scores in decreasing order.
+With the release of this feature, you can now sort the hybrid search results by providing the `sort` criteria in the search request. For more information on the sort criteria, see [sort results](https://opensearch.org/docs/latest/search-plugins/searching-data/sort/).
```

Comment:
Not sure we need to say "with release", just "You can apply sorting to hybrid query results by providing the `sort` criteria in the search request."